pkg/controllers/management/rbac/legacy_ownerrefs_cleaner.go

```diff
@@ -0,0 +1,214 @@
+package rbac
+
+import (
+	"context"
+
+	typesv1 "github.com/rancher/rancher/pkg/generated/norman/core/v1"
+	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
+	v1 "github.com/rancher/rancher/pkg/generated/norman/rbac.authorization.k8s.io/v1"
+	namespaces "github.com/rancher/rancher/pkg/namespace"
+	"github.com/rancher/rancher/pkg/types/config"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/util/retry"
+)
+
+const (
+	roleOwnerRefCorrected        = "auth.management.cattle.io/role-owner-ref-corrected"
+	roleBindingOwnerRefCorrected = "auth.management.cattle.io/role-binding-owner-ref-corrected"
```

Comment:
```suggestion
	ownerRefCorrected        = "auth.management.cattle.io/owner-ref-updated"
```
I think we should just use a single label and swap corrected for updated.